src/coprocessor/mod.rs

```diff
@@ -36,14 +47,14 @@ quick_error! {
             description("region related failure")
             display("region {:?}", err)
         }
-        Locked(l: LockInfo) {
+        Locked(l: kvrpcpb::LockInfo) {
             description("key is locked")
             display("locked {:?}", l)
         }
         Outdated(elapsed: Duration, tag: &'static str) {
             description("request is outdated")
         }
-        Full(allow: usize) {
+        Full {
```

Comment:
Why not keep the `allow`?

Reply:
There are several reasons,
- To keep similar error response as in Storage read pools. 
- I think the allowed size can be easily known according to configurations. `allow` is not essential.
- According to current implementations, we have to keep `allow` at the beginning of the function, which is some what wired because it is only used in errors.